helm/idol-omnigroupserver/templates/config.yaml

```
@@ -8,14 +8,22 @@
 # The information contained herein is subject to change without notice.
 #
 # END COPYRIGHT NOTICE
-{{- if not .Values.existingConfigMap }}
+
+{{- include "idol-library.aciserver.defaultcfg" (dict 
+    "root" .
+    "component" .Values
+    "componentName" (trimPrefix "idol-" .Values.name)
+) }}
+
+---
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Values.name }}-default-cfg
-  labels: {{- include "idol-library.labels" . | nindent 4 }}
+  name: omnigroupserver-oauth-startup-scripts-cm
```

```suggestion
  name: {{ .Values.name }}-oauth-startup-scripts-cm
```
Avoid using hardcoded resource names

---

[eaee33f](https://github.com/opentext-idol/idol-containers-toolkit/pull/31/commits/eaee33f41850e811f1221d6b2d6d9f345bf1821b)